spotrec.py

```diff
@@ -184,7 +189,7 @@ class Spotify:
     playbackstatus_paused = "Paused"
 
     def __init__(self):
-        dbus.mainloop.glib.DBusGMainLoop(set_as_default=True)
+        self.glibloop = dbus.mainloop.glib.DBusGMainLoop(set_as_default=True)
```

Comment:
Is there a reason the glibloop is stored in a variable now or can this be removed?

Reply:
Yes, later on you reference it in quit_glib_loop(). My linter said this variable is not available.

Reply:
The variable self.glibloop is set here:
https://github.com/Bleuzen/SpotRec/blob/a9706601b96f6871576b872c4f21974b67cbd487/spotrec.py#L216

In quit_glib_loop(), this is called:
`self.glibloop.quit()`

This works because the GLib.MainLoop (set in DBusListenerThread) has a quit function.

The DBusGMainLoop does not even have a quit() function, so this wont work.

Setting self.glibloop here sets it to the wrong value.
It still works, because it is overwritten shortly after, but it is still wrong.

An alternative to make the linter happy should be found.

Reply:
Ah, sorry. Will fix.

Reply:
Updated.